app/src/main/java/org/simple/clinic/home/patients/PatientsEffect.kt

```diff
@@ -29,3 +29,7 @@ object LoadNumberOfPatientsRegistered: PatientsEffect()
 object OpenTrainingVideo: PatientsEffect()
 
 object LoadInfoForShowingAppUpdateMessage: PatientsEffect()
+
+object TouchAppUpdateShownAtTime: PatientsEffect()
```

Comment:
Do we need `Touch` prefix here?

Reply:
I named this based on the Unix [`touch`](https://www.computerhope.com/unix/utouch.htm) command since that's essentially what we are doing. Couldn't think of a better name. 😅

Reply:
TIL